### PR TITLE
Update FixBookControl.java

### PR DESCRIPTION
**Pull request by behlsimar**

### CODE REVIEW

**Comment by ghost** [comment]
ok.. all good

File: src/FixBookControl.java [hunk 1]
```diff
@@ -61,7 +63,7 @@ public void scanningComplete() {
 		if (!state.equals(CONTROL_STATE.READY)) {
 			throw new RuntimeException("FixBookControl: cannot call scanningComplete except in READY state");
 		}	
-		ui.setState(FixBookUI.UI_STATE.COMPLETED);		
+		ui.setState(FixBookUi.UI_STATE.COMPLETED); \\Changed FixBookUI to FixBookUi
```
Comment (ghost) [inline src/FixBookControl.java:66]: This is good static review

**Review by ghost — COMMENTED** [review]
everything is good

**Comment by sushant8991** [comment]
i forgot to give permission of write access to everyone so that's why I am merging the file

**Pull request merged**